designer/ui_main_window.ui

```diff
@@ -4360,6 +4361,14 @@
     <string>Result Viewer</string>
    </property>
   </action>
+  <action name="actionOpenSum">
+   <property name="text">
+    <string>Open Sum</string>
```

Comment:
@wdzhou   I don't like reinventing the wheel. Look at line [4183](https://github.com/neutrons/reflectivity_ui/blob/MR8_Merge_Files/designer/ui_main_window.ui#L4183) in this file.

Also, it looks lie "Open Sum" is [implemented in the old quicknxs](https://github.com/aglavic/quicknxs/blob/master/quicknxs/main_gui.py#L1195). We should see how much we can copy/paste.

Reply:
@jmborr I am not sure whether the old wheel has the same spec and whether it can work with the new engine.  There must be some reason why QuickNxs and QuickNxs2 are from two separate repositories.